### ISSUE TITLE
Add TokenIdentifier type(s)

### ISSUE THREAD

**Issue by janniksam** [gfusee/SpaceKit#22]
![image](https://github.com/user-attachments/assets/0e2703c5-0039-4cd9-a9d2-459a42fca17f)

**Comment by gfusee** [comment]
Good point, in the meantime you can use Buffer as a workaround. In the Rust SDK a TokenIdentifier is just a wrapper around a ManagedBuffer

**Comment by gfusee** [comment]
Fixed in #95
